front/app/containers/UserCustomFields/hooks/useUserCustomField.ts

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
```

Comment:
Duplicating a lot of these hooks for now, will remove the ones in the module once I've moved all the admin components